polynote-frontend/polynote/ui.js

```diff
@@ -1791,6 +1794,7 @@ export class MainUI extends EventTarget {
                 window.history.pushState({notebook: tab.name}, `${tab.name.split(/\//g).pop()} | Polynote`, `/notebook/${tab.name}`);
                 this.currentNotebookPath = tab.name;
                 this.currentNotebook = this.tabUI.getTab(tab.name).content.notebook.cellsUI;
+                this.currentNotebook.notebookUI.cellUI.forceLayout(evt)
```

Comment:
I ran into what @kedargsm reported earlier today where the text cells were all blank after switching notebook tabs. This solves it, though I'm not sure why it was only happening some of the time.

Reply:
I've seen that too in my own branch... I think it dates back to the last monaco update (#196). They must be disposing of non-visible stuff for performance reasons?